libs/wingsdk/src/target-sim/queue.ts

```
@@ -18,12 +18,14 @@ import { convertBetweenHandlers } from "../utils/convert";
  */
 export class Queue extends cloud.Queue implements ISimulatorResource {
   private readonly timeout: std.Duration;
+  private readonly retentionPeriod: std.Duration;
   private readonly subscribers: QueueSubscriber[];
   private readonly initialMessages: string[] = [];
   constructor(scope: Construct, id: string, props: cloud.QueueProps = {}) {
     super(scope, id, props);
 
     this.timeout = props.timeout ?? std.Duration.fromSeconds(10);
+    this.retentionPeriod = props.retentionPeriod ?? std.Duration.fromHours(1);
```

Should we throw an exception here if the retain period is less than timeout?